flexmeasures/api/v3_0/tests/test_accounts_api.py

```diff
@@ -0,0 +1,68 @@
+from flask import url_for
+import pytest
+
+from flexmeasures.data.services.users import find_user_by_email
+from flexmeasures.api.tests.utils import get_auth_token
+
+
+def test_get_accounts_missing_auth(client):
+    """
+    Attempt to get accounts with missing auth.
+    """
+    # the case without auth: authentication will fail
+    headers = {"content-type": "application/json"}
+    get_accounts_response = client.get(url_for("AccountAPI:index"), headers=headers)
+    print("Server responded with:\n%s" % get_accounts_response.data)
+    assert get_accounts_response.status_code == 401
+
+
+@pytest.mark.parametrize("as_admin", [True, False])
+def test_get_accounts(client, setup_api_test_data, as_admin):
+    """
+    Get accounts
+    """
+    headers = {"content-type": "application/json"}
```

Comment:
Maybe the headers and authorization can be done in a separate function since it's being reused.